packages/ui-kit/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, ReactNode, InputHTMLAttributes } from "react";
+import clsx from "clsx";
+import { nanoid } from "nanoid";
+
+import styles from "./Radio.module.css";
+import { CheckIcon } from "../icons";
+import { Label } from "../Label";
+
+export interface RadioProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, "className"> {
+  label?: string | ReactNode;
+  className?: {
```

Comment:
Also I'd consider making a type for it like `type ClassNames<TKeys> = Partial<Record<TKeys, string>>` and then do 
```
classNames?: ClassNames<"container" | "inputContainer" ...etc >

Reply:
Good idea! This helper added and used in related components.